README.md

```diff
@@ -41,6 +41,10 @@ How to get it running?
    
    Or using Eclipse by right click `Structures > Run As > Maven Build ...`
 
+Build Status
+---------------------------
+[![Build Status](https://travis-ci.org/JainTarunJain/Structures.svg)](https://travis-ci.org/JainTarunJain/Structures)
+
```

Comment:
I will change the address of the Build status icon link, to my user-name in the repository.